_topic_map.yml

```diff
@@ -2032,6 +2032,8 @@ Name: API reference
 Dir: rest_api
 Distros: openshift-enterprise,openshift-origin
 Topics:
+- Name: Editing kubelet log level verbosity and gathering its logs
```

Comment:
Maybe "its" could be dropped here? So just "Editing kubelet log level verbosity and gathering logs" (unless that's too generic).